lib/tasks/valid_test_data_generator.rb

```diff
@@ -104,4 +104,24 @@ def weighted_choice(selection:, odds:)
       }.flatten.sample
     end
   end
+
+  class AmbitionSpecificPopulater < LeadProviderPopulater
+    class << self
+      def call(name:, total_schools: 3, participants_per_school: 3000)
+        new(name: name).call(total_schools: total_schools, participants_per_school: participants_per_school)
+      end
+    end
```

Comment:
If it is ambition specific we could just specify default for all params and not pass them in the rake task I guess